modules/test/playwright/helpers/json-web-services/JSONWebServicesUserApiHelper.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {liferayConfig} from '../../liferay.config';
+import {ApiHelpers} from '../ApiHelpers';
+
+export class JSONWebServicesUserApiHelper {
+	readonly apiHelpers: ApiHelpers;
+	readonly basePath: string;
+
+	constructor(apiHelpers: ApiHelpers) {
+		this.apiHelpers = apiHelpers;
+		this.basePath = '/api/jsonws/user';
+	}
+
+	async assignUsersToSite(groupId: string, userId: string): Promise<void> {
+		const urlSearchParams = new URLSearchParams();
+
+		urlSearchParams.append('groupId', groupId);
+		urlSearchParams.append('userId', userId);
+
+		return this.apiHelpers.post(
+			`${liferayConfig.environment.baseUrl}${this.basePath}/add-group-users/group-id/${groupId}/user-ids/${userId}`,
+			{
+				data: urlSearchParams.toString(),
+				failOnStatusCode: true,
+				headers: await this.apiHelpers.getJSONWebServicesHeaders(),
+			}
+		);
+	}
+
+	async assignUsersToRole(roleId: string, userIds: string[]): Promise<void> {
+		const urlSearchParams = new URLSearchParams();
+
+		urlSearchParams.append('roleId', roleId);
+		urlSearchParams.append(
+			'userIds',
+			Array.isArray(userIds) ? userIds.join(',') : userIds
```

Comment:
can you store this in a variable so can be used here and in the post? And, but I'm not sure, because the parameter is string[] is always (for typescript) an array